std_msgs/String name                # name of current instance (empty for meta description)
std_msgs/String type_class_package
std_msgs/String type_class
std_msgs/String base_class_package
std_msgs/String base_class
vigir_generic_params/ParameterSetMsg params
